fluentui_listitem/src/main/java/com/microsoft/fluentui/tokenized/tabItem/TabItem.kt

```diff
@@ -120,7 +122,7 @@ fun TabItem(
                         width = Dimension.preferredWrapContent
                     }
                     .padding(start = 8.dp),
-                style = TextStyle(color = textColor, textAlign = TextAlign.Center),
+                style = TextStyle(color = textColor, textAlign = TextAlign.Center, fontSize = textSize),
```

Comment:
Refer https://github.com/microsoft/fluentui-android/blob/01664c5999ff05c8c8b61be2afca758a7088a3a9/fluentui_notification/src/main/java/com/microsoft/fluentui/tokenized/notification/Banner.kt#L96

to use typograpghy